pyairctrl/airctrl.py

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python3
-from Crypto.Cipher import AES
-from Crypto.Util.Padding import pad, unpad
+from Cryptodome.Cipher import AES
+from Cryptodome.Util.Padding import pad, unpad
```

Comment:
why changing these imports?

Reply:
This is required to run it on Ubuntu with python 3.7.5 and following packages (with crypto in name) installed:
```
ii  python3-asn1crypto           0.24.0-1      all          Fast ASN.1 parser and serializer (Python 3)
ii  python3-crypto               2.6.1-10      amd64        cryptographic algorithms and protocols for Python
ii  python3-cryptography         2.6.1-3.1     amd64        Python library exposing cryptographic recipes and
ii  python3-pycryptodome         3.6.1-2build2 amd64        cryptographic Python library (Python 3)
```
Without this change Crypto.Util.Padding is not found:
```
$ ./airctrl.py 
Traceback (most recent call last):
  File "./airctrl.py", line 5, in <module>
    from Crypto.Util.Padding import pad, unpad
ModuleNotFoundError: No module named 'Crypto.Util.Padding'
```

Are you sure Crypto.Util.Padding is correct with pycryptodome>=3.4.7 installed?

I can move this change to separate PR.

Reply:
`pycrypto` conflicts with `pycryptodome` on your installation. I strongly prefer `pycryptodome` over `pycrypto`. There is no `Cryptodome` package in recent versions of `pycryptodome` and that's why you get build failures from Travis CI.

Reply:
#27